apps/filebrowser/src/filebrowser/views_test.py

```diff
@@ -51,6 +54,7 @@
 from desktop.lib.test_utils import grant_access, add_to_group, add_permission, remove_from_group
 from desktop.lib.view_util import location_to_url
 from desktop.conf import is_oozie_enabled, RAZ
+
```

Comment:
why new line? are we not following the notion that if we are importing from hue code then those imports will be together?